openapi/components/requestBodies/PostCashierRequest.yaml

```
@@ -15,6 +15,19 @@ properties:
     description: |-
       ID of a strategy to be applied for this request for `amounts` and `customAmount`.
       If this field is not specified, a randomly selected strategy with a matching `filter` value is applied for empty `amounts` and `customAmount`.
+      If there is no matching strategies, default strategy with the following parameters is applied for empty `amounts` and `customAmount`:
```

```suggestion
      If there is no matching strategy, the default strategy with the following parameters is applied for empty `amounts` and `customAmount`:
```
Small improvement suggestion.